Makefile

```diff
@@ -0,0 +1,41 @@
+PREFIX=/usr/local
+DC=dmd
+
+ifeq ($(DC),dmd)
+	DFLAGS=-betterC
+	DEBUGFLAGS=-debug
+endif
+
+ifeq ($(DC),gdc)
+	DEBUGFLAGS=-g
+endif
+
+ifeq ($(DC),ldc)
+	DEBUGFLAGS=-g
+endif
+
+.PHONY: clean install uninstall debug release
+
+default: debug
+
+debug:
+	$(MAKE) ddcpuid DFLAGS="$(DFLAGS) $(DEBUGFLAGS)"
+
+release:
+	$(MAKE) ddcpuid DFLAGS="$(DFLAGS)"
```

Comment:
Looking good! Last change I'd like is appending `-O -release -boundscheck=off` to this line (at least checked on DMD and LDC, dunno about GDC).